CHANGELOG.md

```diff
@@ -2,6 +2,24 @@ faucet-pipeline-core version history
 ====================================
 
 
+v2.0.1
+------
+
+_2025-02-01_
+
+notable changes for end users:
+
+* dropped support for obsolete Node versions; now requires Node v18 or later
```

Comment:
🤔 Is there a more concise (yet elegant) way to express this?

Reply:
Yes, there is:
```
requires Node v18 or later
```

Reply:
I had considered that, but wanted to be explicit about dropping support. Perhaps:

> bumped Node requirement to v18 or later, dropping support for obsolete versions

🤷

Reply:
Okay, works for me :+1: (I would say that "requires version xy" in a changelog implies that this drops support for older versions.)